src/Microsoft.Dnx.ApplicationHost/Program.cs

```diff
@@ -259,7 +247,33 @@ private Task<int> ExecuteMain(DefaultHost host, string applicationName, string[]
                 return Task.FromResult(1);
             }
 
-            return EntryPointExecutor.Execute(assembly, args, host.ServiceProvider);
+            return EntryPointExecutor.Execute(assembly, args, host.ServiceProvider)
+                        .ContinueWith(t =>
+                        {
+                            t.Exception?.Handle(e =>
```

Comment:
Yes, but as per MSDN `Handle()` does:  _if any exceptions went unhandled, all unhandled exceptions will be put into a new AggregateException which will be thrown_ - so just I want to be sure we don't start wrapping exceptions accidently...